CustomApps/lyrics-plus/Utils.js

```diff
@@ -1,10 +1,11 @@
 class Utils {
     static addQueueListener(callback) {
-        Spicetify.Player.origin2.state.addExtendedStatusListener(callback);
+        Spicetify.Player.origin2.state.addQueueListener(callback);
     }
 
     static removeQueueListener(callback) {
-        Spicetify.Player.origin2.state.removeExtendedStatusListener(callback);
+        Spicetify.Player.origin2.state.queueListeners =
+          Spicetify.Player.origin2.state.queueListeners.filter(v => v == callback);
```

Comment:
I think this should be 
`Spicetify.Player.origin2.state.queueListeners = Spicetify.Player.origin2.state.queueListeners.filter(v => v != callback);`
As we are removing that listener.

Reply:
Correct! Good catch!